inst/extdata/simple20x20/segments.csv

```diff
@@ -0,0 +1,5 @@
+"1","2","50; 1950; 100; 1950; 150; 1950; 200; 1950; 250; 1950; 300; 1950; 350; 1950; 400; 1950; 450; 1950; 500; 1950; 550; 1950; 600; 1950; 650; 1950; 700; 1950; 750; 1950; 800; 1950; 850; 1950; 900; 1950; 950; 1950"
```

Comment:
Should be just:

```
1,2,50;1950;100;1950;150;...950;1950
```

The parser is not a fully blown CSV parser, but mostly just basic "split on separator" parser (for simplicity and perhaps speed).